html/files/knowledge_panel_test.json

```diff
@@ -0,0 +1,100 @@
+{
+    "doyouknow_brands_nutella_423": {
+        "type": "doyouknow",
+        "level": "trivia",
+        "topics": [
+            "ingredients"
+        ],
+        "title": "Do you know why Nutella contains hazelnuts?",
+        "subtitle": "It all started after the second world war...",
+        "elements": [
+            {
+                "element_type": "text",
+                "element": {
+                    "html": "Cocoa beans were expensive and hard to come by after the second world war, so in Piedmont (Italy) where Pietro Ferrero created Nutella, they were replaced with hazelnuts to make <em>gianduja</em>, a mix of hazelnut paste and chocolate."
+                }
+            },
+            {
+                "element_type": "image",
+                "element": {
+                    "url": "https://static.openfoodfacts.org/images/attributes/contains-nuts.png",
+                    "width": 192,
+                    "height": 192
+                }
+            }            
+        ]
+    },
+    "ecoscore": {
+        "type": "ecoscore",
+        "level": "info",
+        "topics": [
+            "environment"
+        ],
+        "title": "Eco-Score D",
+        "subtitle": "High environmental impact",
+        "grade": "d",
+        "icon_url": "https://static.openfoodfacts.org/images/attributes/ecoscore-d.svg",
+        "elements": [
+            {
+                "element_type": "text",
+                "element": {
+                    "text_type": "summary",
+                    "html": "The <a href=\"https://world.openfoodfacts.org/eco-score-the-environmental-impact-of-food-products\">Eco-Score</a> is an experimental score that summarizes the environmental impacts of food products."    
+                }
+            },
+            {
+                "element_type": "text",
+                "element": {
+                    "text_type": "note",
+                    "html": "The Eco-Score was initially developped for France and it is being extended to other European countries. The Eco-Score formula is subject to change as it is regularly improved to make it more precise and better suited to each country."    
+                }
+            },
+            {
+                "element_type": "panel",
+                "element": {
+                    "panel_id": "ecoscore_lca"
+                }  
+            },
+            {
+                "element_type": "text",
+                "element": {
+                    "text_type": "warning",
+                    "html": "<strong>Warning: some information necessary to calculate the Eco-Score with precision is not provided (see the details of the calculation below). You can edit the product to add the missing information.<br><br>If you are the manufacturer of this product, you can send us the information with our <a href=\"https://world.pro.openfoodfacts.org\">free platform for producers</a>."
+                }
+            }
+        ]
+    },
+    "ecoscore_lca" : {
```

Comment:
in order to distinguish that this panel doesn't belong to root, should we add a parent_id (as you suggested on the doc), the parent_id for this would be "ecoscore" and the parent_id for others would be "root".

Reply:
Sounds good. I added a parent_panel_id field.